src/main/scala/com/typesafe/sbt/SbtScalariform.scala

```diff
@@ -56,15 +57,15 @@ object SbtScalariform extends AutoPlugin {
 
   import autoImport._
 
-  override lazy val projectSettings = scalariformSettings
+  //override lazy val projectSettings = defaultScalariformSettings
```

Comment:
Was meant as hint --- in either cases `scalariformSettings` and `defaultScalariformSettings ` --- that this was the potential cause for https://github.com/daniel-trinh/sbt-scalariform/issues/15.